calcom/cal.com
# Session Timeout

## What does this PR do?

Implements Session Timeout setting.

https://user-images.githubusercontent.com/467258/212155421-f82320d2-9be2-45da-a449-4bd198dbe4ad.mov

Closes #6380 

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

Go to Settings > Security > Session and set a session timeout of 5, 10 or 15 minutes. Go back to the app and wait the selected time to see the app logout.
